draft-ietf-quic-http.md

```diff
@@ -1071,7 +1071,8 @@ When a server receives this frame, it aborts sending the response for the
 identified server push.  If the server has not yet started to send the server
 push, it can use the receipt of a CANCEL_PUSH frame to avoid opening a push
 stream.  If the push stream has been opened by the server, the server SHOULD
```

Comment:
"has been opened" is now a bit odd without any qualification, since it's possible it's been opened and subsequently closed.

How about "If the push stream is open, the server SHOULD abruptly terminate that stream."

Reply:
The logic there is that, even if the stream is closed (FIN sent), there might still be value in sending a RESET_STREAM to abort any retransmissions.